gallery/src/components/NxMultiFileUpload/NxMultiFileUploadPage.tsx

```diff
@@ -0,0 +1,219 @@
+ */
+import React from 'react';
+import { NxCode, NxH3, NxP, NxTable, NxTextLink, NxTile } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxMultiFileUploadExample from './NxMultiFileUploadExample';
+import NxMultiFileUploadComplexExample from './NxMultiFileUploadComplexExample';
+import NxMultiFileUploadDisabledExample from './NxMultiFileUploadDisabledExample';
+
+const nxMultiFileUploadCode = require('./NxMultiFileUploadExample?raw');
+const nxMultiFileUploadComplexCode = require('./NxMultiFileUploadComplexExample?raw');
+const nxMultiFileUploadDisabledCode = require('./NxMultiFileUploadDisabledExample?raw');
+
+const NxTextInputPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        <NxCode>NxMultiFileUpload</NxCode> is an RSC user interface for a file-selection form field
+        (an <NxCode>&lt;input type="file"&gt;</NxCode>). Multiple files can be selected with multiple calls
+        to the <NxCode>onChange</NxCode> prop. Any selected files are displayed with the name and size of the
+        file, along with a button to remove the selection.
+      </NxP>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Props</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Name</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Default</NxTable.Cell>
+              <NxTable.Cell>Description</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell>files</NxTable.Cell>
+              <NxTable.Cell>FileList | null</NxTable.Cell>
+              <NxTable.Cell>true</NxTable.Cell>
+              <NxTable.Cell/>
+              <NxTable.Cell>
+                The{' '}
+                <NxCode>
+                  <NxTextLink href="https://developer.mozilla.org/en-US/docs/Web/API/FileList" external>
+                    FileList
+                  </NxTextLink>
+                </NxCode>
+                {' '}representing the user's current file selections, if any. Typically will be null before
+                the user makes a selection.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>onChange</NxTable.Cell>
+              <NxTable.Cell>(FileList | null) =&gt; void</NxTable.Cell>
+              <NxTable.Cell>true</NxTable.Cell>
+              <NxTable.Cell/>
+              <NxTable.Cell>
+                A callback which is called whenever the user updates their file selections. The new selection
+                {' '}/ selections, in the form of a <NxCode>FileList</NxCode>, is passed as the parameter, and
```

Comment:
Are spaces around the slash actually appropriate here? I didn't think they were.  Double check with a tech writer

Reply:
> Are spaces around the slash actually appropriate here? I didn't think they were. Double check with a tech writer

Or consider "selection(s)" instead

Reply:
> > Are spaces around the slash actually appropriate here? I didn't think they were. Double check with a tech writer
> 
> Or consider "selection(s)" instead

I changed the wording for onChange, which makes unnecessary now: [8a3054c](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/8a3054ce1818850deb00183b7c0f9d38fe861437)

I did check with Lisa though, and she confirmed no spaces between the slash, and that selection(s) would have been better in this case